app/controllers/users/registrations_controller.rb

```diff
@@ -50,4 +51,12 @@ def after_sign_up_path_for(resource)
   def after_inactive_sign_up_path_for(resource)
     super(resource)
   end
+
+  def send_signup_emails
```

Comment:
no tests

Reply:
send email from model
models have callbacks AND WE HAVE DISCUSSED IT PREVIOUSLY